db/migrate/20180710153043_add_foreign_key.rb

```diff
@@ -0,0 +1,10 @@
+class AddForeignKey < ActiveRecord::Migration[5.2]
```

Comment:
The migration is OK, but we have one issue, the name.
I know that you added foreign keys but I think we should make the names more descriptive. Today is okay, because we have only 2 or 3 models, but later we can have a lot more and the more descriptive the name is, better.

Reply:
The other think is that we are not checking for nullity in the database, I think we should add `null: false` to those columns